tools/qor_trends.py

```
@@ -338,39 +465,77 @@ def main() -> None:
-            + (f" in domain '{args.domain}'" if args.domain else "")
-            + ".\n"
-            f"Check that the design name matches exactly what orchestrators recorded\n"
-            f"in memory/<domain>/experiences.jsonl (field: 'design_name').",
-            file=sys.stderr,
-        )
+        if not all_design_records:
+            print(
+                f"No runs found for design '{args.design}'"
+                + (f" in domain '{args.domain}'" if args.domain else "")
+                + ".\n"
+                f"Check that the design name matches exactly what orchestrators recorded\n"
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)